src/model/src/ASX_DATA_MOD.F

```diff
@@ -716,33 +702,7 @@ Subroutine INIT_MET ( JDATE, JTIME )
             Grid_Data%BSLP    = 0.0
          End If
 
-!> ccccccccccccccccccccc canopy shade option!ccccccccccccccccccccc (Wei Li)
```

Comment:
@btang1 Please do not remove any of my canopy mods to the ASX_DATA_MOD.F.  Please revert to include all my canopy changes in updated ASX_DATA_MOD.

Reply:
I have made updates, move back all canopy inline for ASX_DATA_MOD